src/main/java/org/killbill/billing/plugin/adyen/client/model/PurchaseResult.java

```diff
@@ -39,10 +39,11 @@
     private final List<PaymentServiceProviderErrorCodes> errorCodes;
     private final String paymentInternalRef;
     private final AdyenCallErrorStatus adyenCallErrorStatus;
+    private final Map<String, String> additionalData;
 
     public PurchaseResult(final String paymentInternalRef,
                           final AdyenCallErrorStatus adyenCallErrorStatus) {
-        this(Optional.<PaymentServiceProviderResult>absent(), null, null, null, null, null, ImmutableList.<PaymentServiceProviderErrorCodes>of(), paymentInternalRef, null, null, adyenCallErrorStatus);
+        this(Optional.<PaymentServiceProviderResult>absent(), null, null, null, null, null, ImmutableList.<PaymentServiceProviderErrorCodes>of(), paymentInternalRef, null, null, adyenCallErrorStatus, null);
```

Comment:
Maybe `ImmutableMap.<String, String>of()` instead of `null` to avoid potential NPEs?